### ISSUE TITLE
UI for Create

### ISSUE THREAD

**Issue by vsl5289** [CSCI-GA-2820-SP24-001/recommendations#1]
### **Feature:** 

**As a** developer
**I need** User Interface (UI) capabilities to create a recommendation from the database
**So that** additional recommendations (elements) can be added to the database

### **Details & Assumptions**

- A database is required
- A functional User Interface (UI) is required
- The recommendation that is created does not already exist in the database
- Recommendation type (e.g., cross-sell, up-sell) can be provided for each recommendation that is created 

### Acceptance Criteria
```
Given the need to input a new recommendation into the database
When I visit the "Home Page"
And I set the "name" to "Feeder"
And I select the "recommendationType" drop-down to "cross-sell"
And I press the "Create" button
Then I should see the message "Success"
When I copy the "Id" field
And I press the "Clear" button
Then the "Id" field should be empty
And the "name" field should be empty
And the "recommendationType" field should be empty
When I paste the "Id" field
And I press the "Retrieve" button
Then I should see the message "Success"
And I should see "Feeder" in the "name" field
And I should see "cross-sell" in the "recommendationType" drop-down
And I should see "recommendationID" in the "Id" field

```

**Comment by loganjmcgowan** [comment]
I'm willing to take ownership of this story, and I will size it as a 3. I also have bandwidth to assist on developing some of the additional stories, because these need to be completed during Sprint 3

**Comment by loganjmcgowan** [comment]
@jason2003120 @vsl5289 if you could review and provide feedback, I would greatly appreciate it, as I want to ensure that I get to the proper level of detail in these stories

**Comment by jason2003120** [comment]
@loganjmcgowan There's 2 attributes missing from the scenario:
1. recommendation name : can be see in "recommendation name" field.
2. recommendation id : can be see in "recommendation name" field.

Also, for the recommendation type, we might use a dropdown in the UI. We usually use dropdowns for limited selections and fields for user typed input.

**Comment by loganjmcgowan** [comment]
Thanks @jason2003120, I made some updates based on your feedback. One question I have is whether the "recommendation name" differs from the "name" column that's in our database. If that is the case, I can add the additional steps for that field as well. I've added in the recommendation ID field, and I have also updated verbiage to reflect the usage of drop-downs. I'm happy to make additional edits to ensure that this story is comprehensive and complete
